### PR TITLE
int64 support

### PR DESCRIPTION
**Pull request by Puppollo**
added support for int64 (time.Duration) type

### CODE REVIEW

File: goflags/goflags.go [hunk 1]
```diff
@@ -42,6 +42,7 @@ func Setup(tag string, tagDefault string) {
 	SetTag(tag)
 	SetTagDefault(tagDefault)
 
+	structtag.ParseMap[reflect.Int64] = reflectInt
```
Comment (felipeweb) [inline goflags/goflags.go]: add tests

File: goenv/goenv.go [hunk 2]
```diff
@@ -26,6 +26,7 @@ func Setup(tag string, tagDefault string) {
 	SetTag(tag)
 	SetTagDefault(tagDefault)
 
+	structtag.ParseMap[reflect.Int64] = reflectInt
```
Comment (felipeweb) [inline goenv/goenv.go]: add tests

**Comment by Puppollo** [comment]
tests added

**Comment by crgimenes** [comment]
Thankyou @Puppollo

**Pull request merged**